time stamp
uint32 num_states
uint32 num_observations
float64[] state
float64[] covariance
float64[] residual
